test/api_integration/apis/saved_queries/saved_queries.js

```diff
@@ -55,11 +55,28 @@ export default function ({ getService }) {
         .send({ description: 'my description' })
         .expect(400));
 
+    it('should return 400 for create saved query with duplicate title', () =>
+      supertest
+        .post(`${SAVED_QUERY_BASE_URL}/_create`)
+        .set(ELASTIC_HTTP_VERSION_HEADER, '1')
+        .send({ ...mockSavedQuery, title: 'my title 2' })
+        .expect(200)
+        .then(() =>
+          supertest
+            .post(`${SAVED_QUERY_BASE_URL}/_create`)
+            .set(ELASTIC_HTTP_VERSION_HEADER, '1')
+            .send({ ...mockSavedQuery, title: 'my title 2' })
+            .expect(400)
+        ));
+
     it('should return 200 for update saved query', () =>
       supertest
         .post(`${SAVED_QUERY_BASE_URL}/_create`)
         .set(ELASTIC_HTTP_VERSION_HEADER, '1')
-        .send(mockSavedQuery)
+        .send({
```

Comment:
I'm not actually sure tbh, possibly a past UX decision? I think it could be worth discussing as a follow up, but I'd prefer to leave this one for now and get some design input after.